thebeetoken/beenest-web
# Refactor pricing

## Description
Corresponds to backend refactor thebeetoken/beenest-backend#655

Note that I scope-creeped myself and decided to remove BuyNow components here; there was some modest overlap in the changes, and we're no longer using those, so seemed like an appropriate time to `git rm`

## How to Test
Verify that pricing appears correct during listing search, and viewing an individual listing.

Which devices did you test on?

## REVIEWERS:
Check against these principles:

### High level
Does this code need to be written?
What are the alternatives?
Will this implementation become a support issue?
How much error margin does this solution have?

### Code
* Does the code follow industry standards?
JS: https://github.com/airbnb/javascript
React: https://github.com/airbnb/javascript/tree/master/react
https://github.com/vasanthk/react-bits
Documentation headers: http://usejsdoc.org/index.html

* Is there duplicated code? Can it be refactored into a shared method?
* Is the code consistent with our project?
* Are there unit tests? Do they test the states?
* Is the person refactoring another developer's code? If possible, did the original developer approve?

### Variables/Naming:
* Would the variable type led to future edge cases?
* Are the variable naming clear? Would the value contain something other than what the name describes.

### Security
* Can this be hacked or abused by the user?

## Further Work
BTC!
